### ISSUE TITLE
low performance

### ISSUE THREAD

**Issue by jianlong-yuan** [feipan664/IntraDA#5]
I follow your step
python train.py --cfg ./configs/advent.yml 
python entropy.py --best_iter BEST_ID --normalize False --lambda1 0.67
python train.py --cfg ./intrada.yml

But i can not reproduce your performance.
For the first stage, i got 41.98 mIoU
For the second stage, i got 44.57 mIoU.

Could you help me?

**Comment by feipanir** [comment]
Hi are you doing  the experiment from GTA5->Cityscapes?  I guess the problem is from the first stage that you got 41.98, which is a bit low than what is expected. I suggest you to train more iterations in the first stage. In our experiments, we got BEST IOU at around 80k iteration or so. And also double check your learning rate, image size, pytorch version, etc.

**Comment by jianlong-yuan** [comment]
I am doing the experiment from GTA5->Cityscapes.
I just use your default settings.
How much miou do you achieve for the first stage?


![image](https://user-images.githubusercontent.com/18137551/86774675-578afb00-c089-11ea-878a-8be6cf8d3151.png)


![image](https://user-images.githubusercontent.com/18137551/86775095-afc1fd00-c089-11ea-8da5-fe01848c7fe6.png)
![image](https://user-images.githubusercontent.com/18137551/86775151-b9e3fb80-c089-11ea-9adc-06f682a4ccf3.png)
![image](https://user-images.githubusercontent.com/18137551/86775174-bfd9dc80-c089-11ea-8c6c-262113222dc5.png)


By the way, pytorch version is 1.5, CUDA10.1

**Comment by feipanir** [comment]
We conduct the experiment on pytorch=0.4.1 and our best miou would be around 43.8 for the inter-da stage. Right now you are using pytoch=1.5. I guess it is probably due to pytorch version but I am not really sure. I will double check it on my local server.

**Comment by jianlong-yuan** [comment]
With pytorch0.4.1, i got 42.4miou at 5w iters. I have finished 10w iters.

**Comment by feipanir** [comment]
It's still a bit low than expected. We usually got 43.8± 0.2. Actually our experiments setup of inter-da keep the same as [ADVENT](https://github.com/valeoai/ADVENT).

**Comment by feipanir** [comment]
A little concern is also from the CUDA version. As I tried on my machine, by using CUDA=9.2 I got the same performance, rather it becomes lower when I switch to CUDA=10.2. So be careful about which cuda version is used currently.

**Comment by xdeng7** [comment]
> With pytorch0.4.1, i got 42.4miou at 5w iters. I have finished 10w iters.

Hi, I was using pytorch1.5.1 and CUDA10.2. I was able to achieve 43.75 for ADVENT, the first stage. And 46.28 for the second stage adaptation.

**Comment by feipanir** [comment]
Hi @xdeng7 thank you for your effort and sharing this information! 
Regards
